### DISCUSSION TITLE
after 136000 steps realistic voice but not understandable language

### DISCUSSION THREAD

**Discussion by corlangerak** [coqui-ai/TTS#2099] [General Q&A]
Hi I have a few quistions regarding the training of a VITS network.
i have been training a VITS model from scratch on a dataset of 1645 recordings in the dutch Language.
After 136000 steps the model is able to reproduce the tonality of the voice of my actor quite accurately, however the spoken text doesn't sound anything like the Dutch language. (at least not to a actual dutch speaking person ;) 

Is this normal after this many steps and should I just continue training? 
I was also wondering if the presence of numbers in a numerical representation within my transcription, could be the cause of my problem?
Or could it be that my dataset is too small?

Here I will also provide some images from tensorboard. As I am quite new to ML I am not really able to interpret them or at which scalar I should look.
All help, thoughts and advice are greatly appreciated!

This is my config file:
[config.txt](https://github.com/coqui-ai/TTS/files/9861506/config.txt)

This is the avg_loss_disc:
![EvalStats_avg_loss_disc](https://user-images.githubusercontent.com/51138739/197813082-905505f9-814f-414c-8877-0aa3683999bf.svg)

This is the avg_loss_0
![EvalStats_avg_loss_0](https://user-images.githubusercontent.com/51138739/197813094-a9c15aeb-508a-4ae8-91d8-c78d842a6eed.svg)

**Comment by erogol** [comment]
If something is not understandable, my initial guess is your char set is not compatible with the language.

**Comment by neurlang** [comment]
check using soxi if your wav is 16bit and if your sample rate is configured well
